id.go

```diff
@@ -54,6 +92,10 @@ func (id *ID) Time() (t time.Time, err error) {
 // Note: This function is unsafe and can change the underlying array
 // Please.. read only!
 func (id *ID) Bytes() (out []byte) {
+	if id == nil {
+		return
+	}
+
 	out = (*id)[:]
```

Comment:
Actually don't need to dereference it, you can operate directly on array pointers.

` out = id[:]`

Reply:
Wait, is this new? I vaguely remember trying this ages ago and the compiler yelling at me.

Reply:
Nope, always worked like that (only with array pointers, not slices)